crates/runtime/build.rs

```diff
@@ -14,10 +14,10 @@ fn main() {
     // Check to see if we are on Linux and the `memfd` feature is
```

Comment:
Update comment here? "check if Linux" -> "check if Unix family" and something about how we can partially support memfd-like operation on other Unices too...